### PR TITLE
Paros functions

### PR DESCRIPTION
**Pull request by jennazenk**
### Added
Tests for Paros contract 

_note_ : this PR contains some changes related to the remove-stemmed-symbol branch because I branched out from there. Finalizing and making a PR of the remove-stemmed-symbol is my next step

@schmidsi if you prefer I can make this PR into remove-stemmed-symbol if that makes more sense to you.

### CODE REVIEW

File: packages/melon.js/lib/olympiad/transactions/registerForCompetition.js [hunk 1]
```diff
@@ -5,93 +5,104 @@ import getVersionContract from '../../version/contracts/getVersionContract';
...
-  ensure(
+    const registeredFundToRegistrant = await olympiadContract.instance.registeredFundToRegistrants.call(
+        {},
+        [fundAddress],
+    );
+    const registrantToRegistrantId = await olympiadContract.instance.registrantToRegistrantIds.call(
+        {},
+        [environment.account.address],
+    );
+    {/*ensure(
```
Comment (schmidsi) [inline packages/melon.js/lib/olympiad/transactions/registerForCompetition.js]: Why is this commented out?

File: packages/melon.js/lib/version/calls/getConfig.js [hunk 2]
```diff
@@ -26,7 +26,9 @@ export type AssetConfig = {
  */
 export type Config = {
   assets: Array<AssetConfig>,
-  complianceAddress: Address,
+  OnlyManagerAddress: Address,
```
Comment (schmidsi) [inline packages/melon.js/lib/version/calls/getConfig.js]: Object keys should start with lowercase

**Comment by schmidsi** [comment]
Looks good for me. Don't worry about branching out of stemmed-symbols. Those changes are also merged in. But I would create a new branch out of master for further work on stemmed symbols and delete the old one.

File: packages/melon.js/lib/version/calls/getConfig.js (hunk 2 above)
Comment (jennazenk) [reply packages/melon.js/lib/version/calls/getConfig.js]: right, will correct in next PR

File: packages/melon.js/lib/olympiad/transactions/registerForCompetition.js (hunk 1 above)
Comment (jennazenk) [reply packages/melon.js/lib/olympiad/transactions/registerForCompetition.js]: for testing needed to be commented out - will uncomment on next PR

**Pull request merged**